vespajlib/src/test/java/com/yahoo/compress/ZstdOuputStreamTest.java

```diff
@@ -0,0 +1,30 @@
+// Copyright Verizon Media. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+package com.yahoo.compress;
+
+import org.junit.jupiter.api.Test;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+
+import static org.junit.jupiter.api.Assertions.assertArrayEquals;
+
+/**
+ * @author bjorncs
+ */
+class ZstdOuputStreamTest {
+
+    @Test
+    void output_stream_compresses_input() throws IOException {
+        byte[] inputData = "The quick brown fox jumps over the lazy dog".getBytes();
+        ByteArrayOutputStream arrayOut = new ByteArrayOutputStream();
+        try (ZstdOuputStream zstdOut = new ZstdOuputStream(arrayOut, 12)) {
+            zstdOut.write(inputData[0]);
+            zstdOut.write(inputData, 1, inputData.length - 1);
+        }
+        byte[] compressedData = arrayOut.toByteArray();
+        ZstdCompressor compressor = new ZstdCompressor();
```

Comment:
You should check that the data has actually been compressed. Simplest is just to check that the output is shorter than the input.

Reply:
I have added a new test case for that. The compressed size is actually larger for this test string.